packages/site/src/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Nimble</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel="icon" href="favicon.ico" sizes="any">
+    <script src="https://unpkg.com/@ni/nimble-components/dist/all-components-bundle.js"></script>
```

Comment:
Should we use the minified bundle instead?

```suggestion
    <script src="https://unpkg.com/@ni/nimble-components/dist/all-components-bundle.min.js"></script>
```